### ISSUE TITLE
Ombi crashing

### ISSUE THREAD

**Issue by MotleyC** [tidusjar/Ombi#2787]
This is what the website says when trying to go to the start page. This is windows 10 64bit

"One or more compilation failures occurred:\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(106,27): error CS1061: 'CustomizationSettings' does not contain a definition for 'HasPresetTheme' and no extension method 'HasPresetTheme' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(110,36): error CS1061: 'CustomizationSettings' does not contain a definition for 'PresetThemeContent' and no extension method 'PresetThemeContent' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(112,47): error CS1061: 'CustomizationSettings' does not contain a definition for 'PresetThemeContent' and no extension method 'PresetThemeContent' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(115,39): error CS1061: 'CustomizationSettings' does not contain a definition for 'PresetThemeContent' and no extension method 'PresetThemeContent' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(115,74): error CS1061: 'CustomizationSettings' does not contain a definition for 'PresetThemeContent' and no extension method 'PresetThemeContent' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(121,41): error CS1061: 'CustomizationSettings' does not contain a definition for 'PresetThemeContent' and no extension method 'PresetThemeContent' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(125,49): error CS1061: 'CustomizationSettings' does not contain a definition for 'CustomCssLink' and no extension method 'CustomCssLink' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)\r\nC:\\Program Files\\Ombi\\Views\\Shared\\_Layout.cshtml(127,57): error CS1061: 'CustomizationSettings' does not contain a definition for 'CustomCssLink' and no extension method 'CustomCssLink' accepting a first argument of type 'CustomizationSettings' could be found (are you missing a using directive or an assembly reference?)"

**Comment by tidusjar** [comment]
Please fill in the GitHub template. 

Where did you download ombi?

**Comment by MotleyC** [comment]
I downloaded it from this website. I have been running OMBI for many many months and I just had a user tell me the page wasn't working. I upgraded to the newest release to see if it fixed the issue before submitting a ticket.

**Comment by tidusjar** [comment]
Did you unblock the zip before extracting?

**Comment by MotleyC** [comment]
Yes. Funny thing is my newsletter still went out last night it just appears
to be the web interface

**Comment by tidusjar** [comment]
Please take a backup of the databases and then delete the ombi folder. Start with a fresh folder.

**Comment by MotleyC** [comment]
Wiping the directory and then putting a fresh copy in place worked. I then put my database files back and everything seemed to be back to normal.
